scss/components/_card.scss

```diff
@@ -99,6 +104,10 @@ $card-margin: $global-margin !default;
     flex: 1 0 auto;
   }
 
+  border-width: 1px;
+  border-style: none inset solid outset;
+  border-color: $light-gray;
```

Comment:
The point is to build off the default two-color palette. My SASS is, in a sense, slight of hand; however, it yields a nice subtle divider between cards.

Do you not like the approach, the design, or both?